Project-Days-Calendar/README.md

```diff
@@ -0,0 +1,111 @@
+# Days calendar project
+
+There are some commemerative days which occur annually, but not on a fixed date.
+
+For example, Ada Lovelace Day happens on the second Tuesday every October.
+
+The _date_ that it occurs is different every year. But it has a fixed pattern.
+
+We have supplied a JSON file ([`data.json`](./data.json)) which contains descriptions of several of these days.
+
+The goal of this project is to present this data usefully to users.
+
+## Requirements
+
+This may be an individual or a team project - your class leaders will tell you which you are doing.
+
+Some of the requirements of this project are only required if you are working in a group of at least a certain size.
+
+You must submit both a link to your GitHub repo, and a link to the deployed website.
+
+Your website must be hosted on the internet, and must be automatically deployed when you merge changes to your GitHub repo.
+
+You must be able to explain every line of code in your project, even ones other people in your group wrote.
+
+### Requirements for everyone
+
+Regardless of your group size, you must:
+
+* Create an HTML page which, when loaded, displays a calendar.
+* The calendar must show every day of the current month, each as a rectangle. Each row of rectangles must show one week. The first column must show Mondays. The first day of the month must be shown in the first row.
+* There must be two links or buttons which, when clicked, switch what is displayed. One button must change the display to the previous month. The other button must change the display to the next month. On repeated clicks, these buttons must keep moving back/forwards in time, one month per click.
```

Comment:
Unless we're asking them to navigate to another page (i.e. different URL), they should be buttons in order to be accessible.

```suggestion
* There must be two buttons which, when clicked, switch what is displayed. One button must change the display to the previous month. The other button must change the display to the next month. On repeated clicks, these buttons must keep moving back/forwards in time, one month per click.
```